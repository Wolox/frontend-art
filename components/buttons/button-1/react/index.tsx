import React from 'react';

import '../styles.scss';

function Button1() {
  return (
    <div className="column center btn-group-1">
      <button className="btn btn-1 m-bottom-2" disabled={true}>
        <div className="btn__content">cancel</div>
      </button>
      <button type="button" className="btn btn-1">
        <div className="btn__content">submit</div>
      </button>
    </div>
  );
}

export default Button1;
