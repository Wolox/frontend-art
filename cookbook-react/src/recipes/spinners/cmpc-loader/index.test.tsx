import React from 'react';
import { render } from '@testing-library/react';

import Loader from './index';

describe('Loader component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Loader simple />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('has simple className if the loader is simple', () => {
    const { getByTestId } = render(<Loader simple />);

    expect(getByTestId('loader')).toHaveClass('simple');
  });
});
