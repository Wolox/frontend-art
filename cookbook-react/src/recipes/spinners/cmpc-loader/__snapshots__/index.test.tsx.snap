// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="loaderContainer simple"
    data-testid="loader"
  >
    <div
      class="ldsRing"
    >
      <div
        class="section section1"
      />
      <div
        class="section section2"
      />
      <div
        class="section section3"
      />
    </div>
  </div>
</DocumentFragment>
`;
